src/mpid/ch4/netmod/ofi/ofi_impl.h

```diff
@@ -425,37 +422,26 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_av_insert(int vni, int rank, void *addrna
     goto fn_exit;
 }
 
-MPL_STATIC_INLINE_PREFIX fi_addr_t MPIDI_OFI_comm_to_phys(MPIR_Comm * comm, int rank)
+MPL_STATIC_INLINE_PREFIX fi_addr_t MPIDI_OFI_av_to_phys(MPIDI_av_entry_t * av)
 {
     if (MPIDI_OFI_ENABLE_SCALABLE_ENDPOINTS) {
-        MPIDI_OFI_addr_t *av = &MPIDI_OFI_AV(MPIDIU_comm_rank_to_av(comm, rank));
-        int ep_num = MPIDI_OFI_av_to_ep(av);
-        int rx_idx = ep_num;
-        return fi_rx_addr(av->dest[0][0], rx_idx, MPIDI_OFI_MAX_ENDPOINTS_BITS);
+        int ep_num = MPIDI_OFI_av_to_ep(&MPIDI_OFI_AV(av));
+        return fi_rx_addr(MPIDI_OFI_AV(av).dest[0][0], ep_num, MPIDI_OFI_MAX_ENDPOINTS_BITS);
     } else {
-        return MPIDI_OFI_COMM_TO_PHYS(comm, rank);
+        return MPIDI_OFI_AV(av).dest[0][0];
     }
 }
 
-MPL_STATIC_INLINE_PREFIX fi_addr_t MPIDI_OFI_av_to_phys(MPIDI_av_entry_t * av)
+MPL_STATIC_INLINE_PREFIX fi_addr_t MPIDI_OFI_to_phys(int grank)
```

Comment:
This isn't actually used from what I can tell. It caught my eye because it is oddly named. Maybe just remove?

Reply:
Sounds good. I'll add a commit.

Reply:
On a second thought, I would like to retain that function. I had this idea that `comm` should be abstracted away from netmod and always use `grank` in netmod and shmmod. That should simplify the interface -- at least the am interface -- a bit. What do you think?

Reply:
I guess I'll remove it after all. That function need fix anyway and we always can add it when we need it. Sorry for going back and forth on it 😄

Reply:
I'm not opposed to the function overall. `MPIDI_OFI_grank_to_phys` would be a better name,  but I think we can add that in a future PR.